controllers/toolchainconfig/configuration_test.go

```diff
@@ -239,20 +239,12 @@ func TestAutomaticApprovalConfig(t *testing.T) {
 		toolchainCfg := newToolchainConfig(cfg, map[string]map[string]string{})
 
 		assert.False(t, toolchainCfg.AutomaticApproval().IsEnabled())
-		assert.Equal(t, 1000, toolchainCfg.AutomaticApproval().MaxNumberOfUsersOverall())
-		assert.Empty(t, toolchainCfg.AutomaticApproval().MaxNumberOfUsersSpecificPerMemberCluster())
-		assert.Equal(t, 80, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdDefault())
-		assert.Empty(t, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdSpecificPerMemberCluster())
 	})
 	t.Run("non-default", func(t *testing.T) {
-		cfg := commonconfig.NewToolchainConfigObjWithReset(t, testconfig.AutomaticApproval().Enabled(true).MaxNumberOfUsers(123, testconfig.PerMemberCluster("member1", 321)).ResourceCapacityThreshold(456, testconfig.PerMemberCluster("member1", 654)))
+		cfg := commonconfig.NewToolchainConfigObjWithReset(t, testconfig.AutomaticApproval().Enabled(false))
 		toolchainCfg := newToolchainConfig(cfg, map[string]map[string]string{})
 
-		assert.True(t, toolchainCfg.AutomaticApproval().IsEnabled())
-		assert.Equal(t, 123, toolchainCfg.AutomaticApproval().MaxNumberOfUsersOverall())
-		assert.Equal(t, cfg.Spec.Host.AutomaticApproval.MaxNumberOfUsers.SpecificPerMemberCluster, toolchainCfg.AutomaticApproval().MaxNumberOfUsersSpecificPerMemberCluster())
-		assert.Equal(t, 456, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdDefault())
-		assert.Equal(t, cfg.Spec.Host.AutomaticApproval.ResourceCapacityThreshold.SpecificPerMemberCluster, toolchainCfg.AutomaticApproval().ResourceCapacityThresholdSpecificPerMemberCluster())
+		assert.False(t, toolchainCfg.AutomaticApproval().IsEnabled())
```

Comment:
You changed the AutomaticApproval from `true` to `false`, however, this test case is for non-default values so we should keep `true` there - by default is `false`

Reply:
good catch, thx! fixed in [20b043177ee](https://github.com/mfrancisc/host-operator/commit/20b043177ee61f6c50a2f22ea5c65106ed5fc2fe)